test/compilable/test15371.d

```diff
@@ -0,0 +1,11 @@
+module compilable.test15371;
+
+import test15371_types : type_test15371;
+
+version(unittest)
+
+void main() {
+    static foreach(m; __traits(allMembers, type_test15371)) {
+        __traits(getMember, m);
```

Comment:
probably a good idea to print them out, and verify the output.

Reply:
What you do mean by print them out? As in to the console?